README.md

```diff
@@ -1,15 +1,12 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Appointment Teacher Backend
```

Comment:
- [ ] A project screenshot would be great for this app!  Please, kindly add one on your readme.  It could be a UML for your models, a screenshot of the API working, or even your swagger docs. 💪